toolboxes/fluid/cases/rigid_body_paper_tests/falling_ellipse/falling_ellipse.cfg

```diff
@@ -1,37 +1,39 @@
-directory=toolboxes/fluid/moving_body/three_sphere_2D/
+
+directory=toolboxes/fluid/new_testcases/falling_ellipse
```

Comment:
why add new_testcases? should be : toolboxes/fluid/moving_body/falling_ellipse

Reply:
fixed